tests/model_hub_tests/torch_tests/hf_transformers_models

```diff
@@ -104,7 +104,8 @@ google/mobilebert-uncased,mobilebert
 google/mobilenet_v1_0.75_192,mobilenet_v1
 google/mt5-base,mt5
 google/owlvit-base-patch32,owlvit
-google/pix2struct-docvqa-base,pix2struct,skip,Load problem
+google/pix2struct-docvqa-base,pix2struct,xfail,Provide example input
+google/pix2struct-ai2d-base,pix2struct,xfail,Could not get TorchScript module by tracing. With exception: bos_token_id has to be defined when no input_ids are provided.
```

Comment:
```suggestion
google/pix2struct-ai2d-base,pix2struct,xfail,Trace error: bos_token_id has to be defined when no input_ids are provided.
```

Reply:
after correcting classes, it fails much later at convert phase, updated this messages